admin/services/prepared-check-sync.service.js

```diff
@@ -1,23 +1,19 @@
-const azureQueueService = require('../services/azure-queue.service')
-const pinGenerationDataService = require('../services/data-access/pin-generation.data.service')
-const featureToggles = require('feature-toggles')
 const sb = require('@azure/service-bus')
 const config = require('../config')
+
 let sbClient
 let sbQueueClient
 let sbQueueSender
 
-const redisPreparedChecks = featureToggles.isFeatureEnabled('_2020Mode')
-
-if (redisPreparedChecks) {
-  sbClient = sb.ServiceBusClient.createFromConnectionString(config.ServiceBus.connectionString)
-  sbQueueClient = sbClient.createQueueClient('check-sync')
-  sbQueueSender = sbQueueClient.createSender()
-}
-
 const preparedCheckSyncService = {}
 
-const redisEnabledBehaviour = async (pupilUrlSlug) => {
+const addMessageToRedis = async (pupilUrlSlug) => {
```

Comment:
shouldn't this be named `addMessageToServiceBus`?

Reply:
Yes! Good catch.  Now updated.